app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/account/home.scala.html

```diff
@@ -105,15 +106,29 @@
 
 @accountHeader = {
   <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">@title</h1>
-  <h2 class="govuk-heading-s govuk-!-font-weight-regular" id="account-details">
-   <span>
-   @messages("account.home.accountName", subscribed.subscribedDetails.makeAccountName())
-   </span>
-   <br>
-   <span>
-   @{messages("account.home.subtitle", subscribed.subscribedDetails.cgtReference.value)}
-   </span>
-  </h2>
+  @govukSummaryList(SummaryList(
```

Comment:
if we add `classes  = "govuk-summary-list--no-border"` to the component then it will remove them